.secrets.baseline

```diff
@@ -128,23 +128,23 @@
         "filename": "deploy/template.yaml",
         "hashed_secret": "19489ea41acf55b1e67d187515d63eb5dfe90fdb",
         "is_verified": false,
-        "line_number": 389
+        "line_number": 392
```

Comment:
To exlcude the secrets, please add them to the array in the baseline file so it doesn't need to be updated every time we add or remove a newline, like we did in the check-hmrc-api repo

https://github.com/govuk-one-login/ipv-cri-check-hmrc-api/blob/ed3077bb986fdaa318c902d6815f645caa82f686/.secrets.baseline#L113-L126